2022-03-25 13:45:21,209 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-25 13:45:21,209 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-25 13:45:21,209 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-25 13:45:21,209 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-25 13:45:21,209 INFO    MainThread:8658 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_134521-2qahjqme/logs/debug.log
2022-03-25 13:45:21,210 INFO    MainThread:8658 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_134521-2qahjqme/logs/debug-internal.log
2022-03-25 13:45:21,210 INFO    MainThread:8658 [wandb_init.py:init():439] calling init triggers
2022-03-25 13:45:21,210 INFO    MainThread:8658 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-25 13:45:21,210 INFO    MainThread:8658 [wandb_init.py:init():458] re-initializing run, found existing run on stack: v6vle3pj
2022-03-25 13:45:21,212 INFO    MainThread:8658 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/v6vle3pj
2022-03-25 13:45:21,213 INFO    MainThread:8658 [jupyter.py:save_history():429] not saving jupyter history
2022-03-25 13:45:21,213 INFO    MainThread:8658 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-25 13:45:21,213 INFO    MainThread:8658 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-25 13:45:21,213 INFO    MainThread:8658 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-25 13:45:21,226 INFO    MainThread:8658 [wandb_run.py:_restore():1674] restore
2022-03-25 13:45:21,454 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-25 13:45:21,739 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-25 13:45:22,565 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 123333
}

2022-03-25 13:45:22,675 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 123333
}

2022-03-25 13:45:22,806 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 123333
}

2022-03-25 13:45:22,915 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 123333
  total_bytes: 123333
}

2022-03-25 13:45:23,023 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 123333
  total_bytes: 123333
}

2022-03-25 13:45:23,135 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 123333
  total_bytes: 123333
}

2022-03-25 13:45:23,245 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 123333
  total_bytes: 123333
}

2022-03-25 13:45:23,353 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 123333
  total_bytes: 123333
}

2022-03-25 13:45:23,461 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 123333
  total_bytes: 123333
}

2022-03-25 13:45:23,570 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 123333
  total_bytes: 123333
}

2022-03-25 13:45:24,817 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 123333
  total_bytes: 123333
}

2022-03-25 13:45:25,283 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 123333
  total_bytes: 123333
}
local_info {
}

2022-03-25 13:45:26,446 INFO    MainThread:8658 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-25 13:45:26,448 INFO    MainThread:8658 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-25 13:45:26,451 INFO    MainThread:8658 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-25 13:45:26,452 INFO    MainThread:8658 [wandb_init.py:init():492] starting backend
2022-03-25 13:45:26,452 INFO    MainThread:8658 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-03-25 13:45:26,455 INFO    MainThread:8658 [backend.py:ensure_launched():219] starting backend process...
2022-03-25 13:45:26,477 INFO    MainThread:8658 [backend.py:ensure_launched():224] started backend process with pid: 9348
2022-03-25 13:45:26,479 INFO    MainThread:8658 [wandb_init.py:init():501] backend started and connected
2022-03-25 13:45:26,497 INFO    MainThread:8658 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-25 13:45:26,497 INFO    MainThread:8658 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-25 13:45:26,497 INFO    MainThread:8658 [wandb_init.py:init():565] updated telemetry
2022-03-25 13:45:26,533 INFO    MainThread:8658 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-25 13:45:27,593 INFO    MainThread:8658 [wandb_run.py:_on_init():1759] communicating current version
2022-03-25 13:45:27,651 INFO    MainThread:8658 [wandb_run.py:_on_init():1763] got version response 
2022-03-25 13:45:27,652 INFO    MainThread:8658 [wandb_init.py:init():625] starting run threads in backend
2022-03-25 13:45:29,925 INFO    MainThread:8658 [wandb_run.py:_console_start():1733] atexit reg
2022-03-25 13:45:29,926 INFO    MainThread:8658 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-25 13:45:29,926 INFO    MainThread:8658 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-25 13:45:29,926 INFO    MainThread:8658 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-25 13:45:29,926 INFO    MainThread:8658 [wandb_init.py:init():664] run started, returning control to user process
2022-03-25 13:45:29,927 INFO    MainThread:8658 [wandb_watch.py:watch():47] Watching
2022-03-25 13:46:01,675 INFO    MainThread:8658 [wandb_config.py:__setitem__():153] config set batch_size = 30 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7faf0294e550>>
2022-03-25 13:46:01,675 INFO    MainThread:8658 [wandb_run.py:_config_callback():992] config_cb batch_size 30 None
2022-03-25 13:46:01,675 INFO    MainThread:8658 [wandb_config.py:__setitem__():153] config set epochs = 10 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7faf0294e550>>
2022-03-25 13:46:01,675 INFO    MainThread:8658 [wandb_run.py:_config_callback():992] config_cb epochs 10 None
2022-03-25 13:46:01,675 INFO    MainThread:8658 [wandb_config.py:__setitem__():153] config set lr = 1.1026909039489844e-05 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7faf0294e550>>
2022-03-25 13:46:01,675 INFO    MainThread:8658 [wandb_run.py:_config_callback():992] config_cb lr 1.1026909039489844e-05 None
2022-03-25 13:46:01,676 INFO    MainThread:8658 [wandb_config.py:__setitem__():153] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7faf0294e550>>
2022-03-25 13:46:01,676 INFO    MainThread:8658 [wandb_run.py:_config_callback():992] config_cb no_cuda False None
2022-03-25 13:46:01,676 INFO    MainThread:8658 [wandb_config.py:__setitem__():153] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7faf0294e550>>
2022-03-25 13:46:01,676 INFO    MainThread:8658 [wandb_run.py:_config_callback():992] config_cb seed 42 None
2022-03-25 13:46:19,536 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-25 13:46:19,537 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-25 13:46:19,537 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-25 13:46:19,537 INFO    MainThread:8658 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-25 13:46:19,537 INFO    MainThread:8658 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_134619-yc3dxfpb/logs/debug.log
2022-03-25 13:46:19,537 INFO    MainThread:8658 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220325_134619-yc3dxfpb/logs/debug-internal.log
2022-03-25 13:46:19,537 INFO    MainThread:8658 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7faedcd0aac0>
2022-03-25 13:46:19,537 INFO    MainThread:8658 [wandb_init.py:init():439] calling init triggers
2022-03-25 13:46:19,537 INFO    MainThread:8658 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-25 13:46:19,537 INFO    MainThread:8658 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 2qahjqme
2022-03-25 13:46:19,539 INFO    MainThread:8658 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/2qahjqme
2022-03-25 13:46:19,539 INFO    MainThread:8658 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-25 13:46:19,541 INFO    MainThread:8658 [wandb_run.py:_restore():1674] restore
2022-03-25 13:46:20,135 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-25 13:46:20,389 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-25 13:46:20,892 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 55709
}

2022-03-25 13:46:21,000 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 55709
}

2022-03-25 13:46:21,109 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 55709
}

2022-03-25 13:46:21,216 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 55709
  total_bytes: 55709
}

2022-03-25 13:46:21,327 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 55709
  total_bytes: 55709
}

2022-03-25 13:46:21,437 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 55709
  total_bytes: 55709
}

2022-03-25 13:46:21,567 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 55709
  total_bytes: 55709
}

2022-03-25 13:46:21,678 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 55709
  total_bytes: 55709
}

2022-03-25 13:46:22,328 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 55709
  total_bytes: 55709
}

2022-03-25 13:46:22,800 INFO    MainThread:8658 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 55709
  total_bytes: 55709
}
local_info {
}

2022-03-25 13:46:23,942 INFO    MainThread:8658 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-25 13:46:23,943 INFO    MainThread:8658 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-25 13:46:23,945 INFO    MainThread:8658 [wandb_run.py:_footer_sync_info():2822] logging synced files
